firefox_ui_tests/resources/cookie_single.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Please put this file into the `cookies` sub folder so we keep those files separate by module.